tests/sls/homu/valid_builders.py

```diff
@@ -0,0 +1,40 @@
+import json
+import subprocess
+import toml
+
+from tests.util import Failure, Success
+
+
+def is_builders_valid(buildbot, homu):
+    for builder in homu:
+        if builder not in buildbot:
+            return False
+    return True
+
+
+def run():
+    homu_cfg = toml.load('/home/servo/homu/cfg.toml')
+    homu_buildbot = homu_cfg['repo']['servo']['buildbot']
+
+    ret = subprocess.run(
+        ['python2', 'tests/sls/homu/get_buildbot_cfg.py'],
```

Comment:
Also, right now the rest of the test suite is agnostic to CWD, so I'd prefer to refer to the `get_buildbot_cfg.py` file differently, likely using `__file__`.